README.md

```diff
@@ -16,3 +16,9 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+The first thing that I would do to start verifying without the code, is to input different sized lists from small medium to large. To add other tests I could input the lists in different ways, such as already sorted or reverse sorted or other set ups like that. I would input a 10 size list a 100 size increasing up to around 1,000,000,000 size and maybe even larger depending on the results. By using sizes of ten it would be easy to analyze if the claim is correct because the claim is linear, so if size 100 input is linearly larger from 10 input and so on we can use that as evidence that the claim is correct. And to measure this i just need to time each run for each input.
+
+In theory i dont believe this time complexity could be correct. I dont believe it could be correct becuase we found that the lower bound time complexity of a comparison based algorithm would have to be Big Omega n log n because it takes linear time to go through the list and log n to compare. Because the theoretical is claiming to be linear it is not following the lower bound that we found in class for comparable sorting algorithms.
```

Comment:
I don't understand what you mean when you say that it takes log n time to compare. How do you know that?

Reply:
i was talking about how over the entire program it would take log n time to total all the compares and divide them accordingly not just one comparison because that would just be constant time.  Is that not right?

Reply:
How do you know that there are log n comparisons? Don't you need more?

Reply:
Maybe im not thinking about it right then.  In my head the comparisons in most cases on a arbitrary sorting algorthm would take log n, is it actually linear because it goes through the list comparing?

Reply:
Well, the argument we talked about in class for the theoretical analysis of the problem got us to n log n comparisons.

Reply:
Because it goes through the list of elements giving us linear plus the comparison itself giving n log n?

Reply:
In class we talked about the average comparison being n log n, is the linear part because in most comparison algorithms its comparing every element?

Reply:
The minimum (not average) number of comparisons is asymptotically bounded by n log n. There's no linear part; this is the complexity for the number of comparisons.

Reply:
So i talked with Ali and i think i understand now.  In the text book they go over the proof of why in most cases except for special examples the complexity is n log n at the best, so if they claim that their comparison based algorithm works in n time then that would not be possible because that is better then n log n